tests/benchdnn/inputs/graph/pattern/bf16/bf16_matmul_bias_post_ops_chain_fusion.json

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
I don't think it's necessary to keep all of these test files. Why do we want to specifically test both `matmul + clamp` and `matmul + elu`?
Please check what's the case used by the customer and if we can add the case to the tests.